src/Time/Units.hs

```diff
@@ -300,7 +301,7 @@ fortnight = time
 {-# INLINE fortnight #-}
 
 -- | Returns the greatest integer not greater than given 'Time'.
-floorRat :: forall (unit :: Rat) b . (Integral b) => Time unit -> b
+floorRat :: forall b (unit :: Rat) . (Integral b) => Time unit -> b
```

Comment:
Oh cool, you ended up doing #130 in this PR. Let's add this to the changelog as well then ^^